packages/volto/src/components/manage/Widgets/SelectWidget.jsx

```diff
@@ -217,6 +230,29 @@ class SelectWidget extends Component {
             : []),
         ];
 
+    if (additionalChoices) {
+      options = [
+        ...(options || []),
+        ...map(additionalChoices, (choice) => ({
```

Comment:
I would start using standard js for these easy things, instead of using lodash. Nowadays we don't need lodash for these simple things like map or the filter a few lines below imho

Reply:
Agree, fixed.